core/src/main/kotlin/org/evomaster/core/problem/rest/seeding/postman/pojos/Url.kt

```diff
@@ -0,0 +1,9 @@
+package org.evomaster.core.problem.rest.seeding.postman.pojos
+
+data class Url (
+        val raw: String,
+        val protocol: String,
+        val host: List<String>,
+        val path: List<String>,
+        val query: List<Query>?
+)
```

Comment:
same as above: if a collection might be missing, best to have it empty than having it nullable

Reply:
Same as previous comment.